aim/web/ui/src/components/Table/DeleteModal.tsx

```diff
@@ -15,16 +15,27 @@ import { processDurationTime } from 'utils/processDurationTime';
 function DeleteModal({
   opened,
   onClose,
-  selectedRows,
+  selectedRows = {},
   onRowSelect,
   deleteRuns,
-}: any): React.FunctionComponentElement<React.ReactNode> {
-  let runsDeleteRequest: any = null;
-  const [data, setData] = React.useState<any[]>([]);
-  const [disabledData, setDisabledData] = React.useState<any[]>([]);
+}: {
+  opened: boolean;
+  onClose: () => void;
+  selectedRows: { [key: string]: any };
+  onRowSelect: ({
+    actionType,
+    data,
+  }: {
+    actionType: 'single' | 'selectAll' | 'removeAll';
+    data?: any;
+  }) => {
+    [key: string]: any;
+  };
+  deleteRuns: (ids: string[]) => void;
+}): React.FunctionComponentElement<React.ReactNode> {
   const tableRef = React.useRef<any>({});
   const disabledTableRef = React.useRef<any>({});
-  const [dateNow, setDateNow] = React.useState(Date.now());
+  let runsDeleteRequest: any = null;
```

Comment:
I think the abort functionality isn't needed here. Mainly we use abort on getting requests.